tests/multiobjective/test_ops.py

```diff
@@ -34,27 +36,47 @@ def generate_test_pop():
     # benchmark.  It only requires a single gene.
     problem = SCHProblem()
     pop = [Individual(genome=np.array(g), problem=problem) for g in range(-2,3)]
-
+    pop.append(pop[-1].clone())
+    
     pop = Individual.evaluate_population(pop)
+    
+    ranks = [
+        3, # [4, 16] Note: Shared single-objective fitness with a higher rank
+        2, # [1, 9]
+        1, # [0, 4]
+        1, # [1, 1]
+        1, # [4, 0]
+        1  # [4, 0] Note: Duplicated fitness
+    ]
```

Comment:
This is great! I think it's good enough for an initial PR.

Wondering if we can think up one or two more more test inputs, though.  I'm not sure what the boundary conditions are that we should check, but I imagine there are a few kinds of bugs that this test would not catch. Maybe we can add a test using an additional problem?  Either a follow-on task for @markcoletti or something @lukepmccombs could add now if he has time.